### DISCUSSION TITLE
Feature request - Portuguese (pt-pt) pre-trained model.

### DISCUSSION THREAD

**Discussion by joaocps** [snakers4/silero-models#145] [Ideas]
## 🚀 Feature
There is a huge lack of Portuguese (pt-pt) templates in all open source speech-to-text tools.

Do you think you can make a pre-trained model available in the future?

**Comment by snakers4** [comment]
Hi @joaocps ,

Since we a the for-profit company, given the current situation and lack of any tangible business in Europe, investing time in any European language STT seems like a bad idea.

As for TTS - if there are datasets, we can do it.

STT requires much more investment to make a model that would be better than know APIs like Google.

Best,
Alex
